scripts/apiStage.sh

```diff
@@ -64,6 +61,13 @@ stageApi()
   $PODMAN rm $NAME
   $PODMAN pull $CONTAINER
 
+  $PODMAN stop -t 0 qpc-redis
+  $PODMAN rm qpc-redis
+  $PODMAN pull registry.redhat.io/rhel9/redis-6:latest
```

Comment:
@infinitewarp Anyway there is a redis image that doesn't require logging in to the customer portal?

<insert comments about licenses and having to create or log into something to be able to use the work>

Reply:
if the image is available on the open registry, you can replace `registry.redhat.io` with `registry.access.redhat.com`.

if this doesn't work, then login is required.

Reply:
This is the image quipucords uses as defined here:
https://github.com/quipucords/quipucords-installer/blob/main/config/quipucords-redis.container
```
Image=registry.redhat.io/rhel9/redis-6:latest
```

Others certainly exist in docker hub, quay, and elsewhere, and you are welcome to change it at your own risk, but the dev team can't support others.

Reply:
I just confirmed and redis repo is not available in the open registry:

```
Error: initializing source docker://registry.access.redhat.com/rhel9/redis-6:latest: reading manifest latest in registry.access.redhat.com/rhel9/redis-6: unsupported: This repo requires terms acceptance and is only available on registry.redhat.io 
```

So, yeah, login is required.

Reply:
fixed by just moving to a docker hosted redis:6-alpine... good enough for development